ide_profiles/xcode/README.md

```diff
@@ -0,0 +1,12 @@
+## Xcode IDE profile
+
+Use this profile to edit the project using Xcode.
+
+To use this profile, open `ExtendedKF.xcodeproj` in this folder.
+
+To build using Xcode, re-generate this project as follows:
+
+1. Remove all files from this directory except of `README.md`
```

Comment:
Yep. But since there is only one file left, I've updated the language to be more clear.